test/CMakeLists.txt

```diff
@@ -6,21 +6,23 @@ include(chimeraFunctions)
 find_package(Boost REQUIRED COMPONENTS python)
 find_package(PythonLibs REQUIRED)
 
-include_directories(SYSTEM
-  ${Boost_INCLUDE_DIRS}
-  ${PYTHON_INCLUDE_DIRS}
-)
-
 add_chimera_binding(
   EXCLUDE_FROM_ALL
   TARGET test_binding
   SOURCES dart-example.cpp
   NAMESPACES dart
 )
+
+target_include_directories(test_binding SYSTEM
+  PRIVATE ${Boost_INCLUDE_DIRS}
+  PRIVATE ${PYTHON_INCLUDE_DIRS}
```

Comment:
Nit: We don't need to specify `PRIVATE` for each item. This would work:
```cmake
target_include_directories(test_binding SYSTEM
  PRIVATE 
    ${Boost_INCLUDE_DIRS}
    ${PYTHON_INCLUDE_DIRS}
)
```